microsetta_interface/templates/home.jinja2

```diff
@@ -24,23 +24,23 @@
         {% endif %}
     </div>
     {% else %}
-    <p>
-        It appears that your email has not yet been verified.
-        Please check your email account (and spam folder) for a verification email
-        from "The Microsetta Initiative" and follow its instructions.
-        This email will come from our authentication service AuthRocket,
-        and use the email address "noreply@loginrocket.com".
+  <p>
+        To continue: Check your email account to obtain your confirmation email and link, which was sent to confirm your account.
+        This email will come from our authentication service, AuthRocket, under the email address "noreply@loginrocket.com".
     </p>
     <p>
-        If you cannot locate the original verification email, have it resent by
-        clicking <a href = "{{ authrocket_url }}/profile">here</a> to view your AuthRocket profile and then
-        clicking the "resend verification email" link.
+        If you cannot locate the original confirmation email, have it resent by
+        clicking the button below to view your AuthRocket profile, and then
+        clicking the "resend verification email" link next to your email address.
     </p>
     <p>
-        <button class = "btn btn-primary" onclick="window.location.replace('{{authrocket_url}}/login?redirect_uri={{endpoint}}/authrocket_callback');">
-            Refresh
-        </button>
+        <a href = "{{ authrocket_url }}/profile"><button class = "btn btn-primary">
```

Comment:
I'm really concerned about whether this introduces a functional change as it removes the `onclick` behavior. What was the intention with this change?